automated install/basic-install.sh

```diff
@@ -885,9 +885,9 @@ getStaticIPv4Settings() {
                 --cancel-label "Exit" \
                 --backtitle "IP information" \
                 --title "FYI: IP Conflict" \
-                --msgbox "\\nIt is possible your router could still try to assign this IP to a device, which would cause a conflict\
-But in most cases the router is smart enough to not do that.\
-If you are worried, either manually set the address, or modify the DHCP reservation pool so it does not include the IP you want.\
+                --msgbox "\\nIt is possible your router could still try to assign this IP to a device, which would cause a conflict. \
+But in most cases the router is smart enough to not do that. \
+If you are worried, either manually set the address, or modify the DHCP reservation pool so it does not include the IP you want. \
 It is also possible to use a DHCP reservation, but if you are going to do that, you might as well set a static address."\
```

Comment:
We could improve readability here, replacing the first dot with a comma and using line breaks to separate each sentence. 

```suggestion
                --msgbox "\\nIt is possible your router could still try to assign this IP to a device, which would cause a conflict, \
but in most cases the router is smart enough to not do that.\
\n\nIf you are worried, either manually set the address, or modify the DHCP reservation pool so it does not include the IP you want.\
\n\nIt is also possible to use a DHCP reservation, but if you are going to do that, you might as well set a static address."\
```